### DISCUSSION TITLE
undefined key `set_davg_zero` is not allowed in strict mode

### DISCUSSION THREAD

**Discussion by ZSHtju** [deepmodeling/deepmd-kit#2393] [Q&A]
I am using the recently released deepmd 2.2.0 installed from conda.
When I using finetune OC_10M.pb, dp cannot train the dataset with the error " undefined key `set_davg_zero` is not allowed in strict mode". I try to delete this key, but it will be automatically added into the input file.

![image](https://user-images.githubusercontent.com/23290538/225193638-b36e4ac9-3a88-4ef1-b23d-e81ea2d3be8f.png)

**Comment by njzjz** [comment]
Fixed in #2397.
